.github/workflows/build-test.yml

```diff
@@ -53,11 +53,3 @@ jobs:
         if: runner.os == 'Linux'
       - run: npm test
         if: runner.os != 'Linux'
-      - name: Set up Ruby 2.6
-        uses: actions/setup-ruby@v1
-        with:
-          ruby-version: 2.6.x
-      - name: Check links with awesome_bot
-        run: |
-          gem install awesome_bot
-          awesome_bot --allow-dupe --allow-redirect --skip-save-results `ls *.md` --white-list gitter.im
```

Comment:
This was part of a previous Travis pipeline that checks links in documentation for 404s - I don't think we want to continue failing builds for this.

Reply:
rather than failing - can the bot show a warning message somewhere/log the 404's or broken links?

Reply:
I mostly made this change to get rid of our pipeline dependency on ruby and a third party extension / github action. I don't know if our reliance on documentation links remaining live / active is something I'd like to fail builds for - like I mentioned - but if we can find an easier way to get build warnings without these dependencies I'd prefer that.